Model/Config/Source/Available/Available.php

```diff
@@ -157,4 +177,92 @@ protected function _isAvailable()
 
         return false;
     }
+
+    public function setDefaults($Paymentmethods)
+    {
+
+        $Magento_paymentmethod_codes = array(
```

Comment:
Dit moet op een andere manier.  Bovendien voer je de setDefaults voor elk betaalmethode uit?!

Reply:
Wordt nu niet meer voor elke betaalmethode uitgevoerd.

Hoe kunnen we dit anders doen?